testsuite/tests/kuadrant/reconciliation/cr_configuration/test_reconcile_limitador.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for Kuadrant sub component - Limitador CR configuration via Kuadrant CR"""
+
+import pytest
+
+pytestmark = [pytest.mark.kuadrant_only]
+
+
+REPLICAS_1 = {"replicas": 1}
+REPLICAS_2 = {"replicas": 2}
+
+
+@pytest.fixture()
+def commit():
+    """Omitting authorization as it is not needed"""
```

Comment:
I do not see how this omits only authorization? Wouldnt it remove also RateLimitPolicy?